exporters/zipkin/zipkin.go

```diff
@@ -36,20 +39,21 @@ const (
 type Exporter struct {
 	url    string
 	client *http.Client
-	logger *log.Logger
+	logger logr.Logger
 
 	stoppedMu sync.RWMutex
 	stopped   bool
 }
 
 var (
-	_ sdktrace.SpanExporter = &Exporter{}
+	_           sdktrace.SpanExporter = &Exporter{}
+	emptyLogger                       = logr.Logger{}
```

Comment:
Don't mix type assertions with var declarations, it becomes hard to read and distinguish.

```suggestion
	_ sdktrace.SpanExporter = &Exporter{}
)

var (
	emptyLogger = logr.Logger{}
```

Reply:
updated